### PR TITLE
hotfix/push notifications language

### PR DESCRIPTION
**Pull request by uiboy**

### CODE REVIEW

File: openbook_notifications/migrations/0013_auto_20190709_1919.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('openbook_notifications', '0012_auto_20190623_1657'),
+    ]
+
```
Comment (uiboy) [inline openbook_notifications/migrations/0013_auto_20190709_1919.py]: Not sure why this migrations showed up, these are not my changes 🤔

Comment (lifenautjoe) [reply openbook_notifications/migrations/0013_auto_20190709_1919.py]: huh what

Comment (lifenautjoe) [reply openbook_notifications/migrations/0013_auto_20190709_1919.py]: don't commit it then 😬

Comment (uiboy) [reply openbook_notifications/migrations/0013_auto_20190709_1919.py]: Yea it was by mistake :( cos i added files in the `openbook_notifications`

**Pull request merged**